lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -427,6 +471,7 @@ public function get_item_schema() {
 				'id'                 => array(
 					'description' => __( 'Unique identifier for the post.', 'default' ),
 					'type'        => 'integer',
+					'context'     => array( 'edit', 'embed' ),
```

Comment:
The context settings here allows each property to show in the edit context (the default) and show when embedded when requesting the parent font family.